tests/v2/test_waku_rln_relay_onchain.nim

```diff
@@ -104,6 +106,31 @@ proc createEthAccount(): Future[(keys.PrivateKey, Address)] {.async.} =
   return (pk, acc)
 
 procSuite "Waku-rln-relay":
+
+  ########################
+  ## Ganache installation
+  ########################
+
+  # We install Ganache
+  let installGanache = startProcess("npm", args = ["install",  "ganache"], options = {poUsePath})
+  debug "Ganache installation completed. Printing install log", returnCode=installGanache.waitForExit(), log=installGanache.outputstream.readAll()
+
+  # We run Ganache daemon
+  # Note that we run directly "node node_modules/ganache/dist/node/cli.js" rather than using "npx ganache", so that the daemon does not spawn in a new child process. 
+  # In this way, we can directly send a SIGINT signal to the corresponding PID to gracefully terminate Ganache without dealing with multiple processes.
```

Comment:
I'd add the description of the argument as well as a link to the ganache documentation https://github.com/trufflesuite/ganache#documentation:
`-p` stands for port, `-l` sets the gas limit, and `-e` sets the default account balance, specified in ether.

Reply:
Indeed! This is a very good point, I completely missed it:

 @s1fr0 instead of using the short version of the flags, can you use the long version (e.g., `--port` instead of `-p`?